### DISCUSSION TITLE
Post UPYOG Related Queries

### DISCUSSION THREAD

**Discussion by niua-upyog** [upyog/UPYOG#36] [Q&A]
Please ask your questions related to UPYOG.

**Comment by haiderali1** [comment]
Hello Team,
We have recieved a list of software that needs to be present before the training 
can we have the installables in windows machine and have an ubuntu machine in the background for cases like kafka

**Comment by rndudhe1808** [comment]
hello team,
i need help about ci-libs library

**Comment by rahulmishraabmindia** [comment]
When I am changing in code structure not reflecting on website please suggest
